docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -146,13 +147,22 @@ If you are creating a cart for a specific customer, pass in the `customer_id` in
 }
 ```
 
-Contained in the response is a `UUID` which we'll use as the `cart_id` in the next request. 
+Contained in the response is a `id` which we'll use as the `cart_id` in the next request: 
+
+**Create Cart Response**
 	
-Next, generate a cart URL and set this cart as the active cart by posting to  to `/carts/{{cart_id}}/redirect_urls`:
+```json
+{
+    "data": {
+        "id": "33608b81-ba34-4ff2-8bab-2771aeab3f73",
+    ...
+}
+```
+Next, generate cart redirect URLs and set this cart as the active cart by sending a request to `/carts/{{cart_id}}/redirect_urls`.
```

Comment:
this needs to be broken into multiple steps/sentences. Each sentence should be followed by their corresponding code sample.